pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
+, enableZlib ? false, zlib
+, enablePng ? false, libpng
+, enableEigen ? false, eigen }:
+
+stdenv.mkDerivation rec {
+  version = "1.1";
+  name = "openmvg-${version}";
+
+  src = fetchgit {
+    url = "https://www.github.com/openmvg/openmvg.git";
+
+    # Tag v1.1
+    rev = "f5ecb48";
+    sha256 = "1di9i7yxnkdvl8lhflynmqw62gaxwv00r1sd7nzzs9qn63g0af0f";
+    fetchSubmodules = true;
+  };
+
+  buildInputs =
+       [ glibc ]
+    ++ lib.optional enableJpeg libjpeg
+    ++ lib.optional enableZlib zlib
+    ++ lib.optional enablePng libpng
+    ++ lib.optional enableEigen eigen
+    ;
+
+  nativeBuildInputs = [ cmake gnumake pkgconfig ];
+
+  configurePhase = ''
+    cmake ./src -DCMAKE_INSTALL_PREFIX=$out -DCMAKE_BUILD_TYPE=Release
+  '';
+  
+  # This can be enabled, but it will exhause virtual memory on most machines.
+  enableParallelBuilding = false;
+
+  hardeningDisable = [ "all" ];
+
+  meta = {
+    description = "A library for computer-vision scientists and targeted fort the Multiple View Geometry community";
```

Comment:
Small typo here with "for"(t). I'll remove this in another PR to avoid an entire recompile of OpenMVG on a single thread (don't want to exhaust your virtual memory on the testing machine).